### PR TITLE
{bio}[goolf/1.4.10] tabix v0.2.6

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar missing patch file? `tabix-0.2.6-zlib.patch`

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb [hunk 1]
```diff
@@ -1,38 +1,66 @@
 # This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
 # Author: Pablo Escobar Lopez
-# Swiss Institute of Bioinformatics
-# Biozentrum - University of Basel
-# Author: Jens Timmerman, Ghent University
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb]: forgotten author: field?

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb [hunk 2]
```diff
@@ -1,38 +1,66 @@
 # This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
 # Author: Pablo Escobar Lopez
-# Swiss Institute of Bioinformatics
-# Biozentrum - University of Basel
-# Author: Jens Timmerman, Ghent University
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
 
-easyblock = 'MakeCp'
+easyblock='MakeCp'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb]: spaces?

File: easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb [hunk 3]
```diff
@@ -1,38 +1,66 @@
...
+prebuildopts += 'cd perl && '
+prebuildopts += 'perl Makefile.PL PREFIX=%(builddir)s && '
+prebuildopts += 'make && make install && '
+prebuildopts += 'cd .. && '
 
 files_to_copy = [
     (["tabix", "bgzip", "tabix.py"], "bin"),
     (["tabix.1"], "man/man1"),
-    "example.gtf.gz",
+    "example.gtf.gz",
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/t/tabix/tabix-0.2.6-goolf-1.4.10.eb]: lgtm overall, trim down unneeded spaces at end of lines (mostly to keep the PRs clean and concise)

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**